### PR TITLE
Fix nightly test issue and add docs build settings for 1.3.x

### LINKED ISSUE #13320: NightlyTestsForBinaries failed
## Description

All builds fail after this [run](http://jenkins.mxnet-ci.amazon-ml.com/job/NightlyTestsForBinaries/job/master/135/):

```
Still waiting to schedule task
Waiting for next available executor on mxnetlinux-cpu
Running on mxnetlinux-cpu_d08jbgdfmd in /home/jenkins_slave/workspace/htlyTestsForBinaries_master-FRAH5QII62EC3VNIIQH2IVTROVD7N2AHQN7XE57SAXHC2VE65MXQ
[Pipeline] {
[Pipeline] sh
[htlyTestsForBinaries_master-FRAH5QII62EC3VNIIQH2IVTROVD7N2AHQN7XE57SAXHC2VE65MXQ] Running shell script
+ echo caught hudson.AbortException: script returned exit code 2
caught hudson.AbortException: script returned exit code 2
```

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Description ##
This PR fixes #13320 and it adds settings for 1.3.0 and 1.3.1.

### CODE REVIEW

**Comment by kalyc** [comment]
@mxnet-label-bot add [pr-awaiting-review, CI, Build, Doc]

File: docs/settings.ini [hunk 1]
```diff
@@ -14,6 +15,34 @@ java_docs = 1
...
+scala_docs = 1
+
+[document_sets_1.3.0]
+clojure_docs = 1
+doxygen_docs = 1
+java_docs = 0
+r_docs = 0
+scala_docs = 1
+
+[document_sets_v1.3.0]
```
Comment (Roshrini) [inline docs/settings.ini]: Why do we have configuration for each version twice?

Comment (aaronmarkham) [reply docs/settings.ini]: CI uses branches.

Comment (lebeg) [reply docs/settings.ini]: Would it be possible to add something like this instead:
```
[document_sets_v1.3.x]
clojure_docs = 1
doxygen_docs = 1
java_docs = 0
r_docs = 0
scala_docs = 1
```

**Comment by lebeg** [comment]
@nswamy

**Comment by aaronmarkham** [comment]
@lebeg Yes, you nailed it. Thanks for catching that. I also just woke up with this realization (yay 4am): that I didn't use the actual branch name since the name format changed with 1.3.
I fixed it now. I'm going to run the full website build with this just to be sure.

**Comment by aaronmarkham** [comment]
Test completed and built all versions including using the setting for 1.3.1.
```
Building v1.3.x...
+ cd v1.3.x/docs
+ make html USE_OPENMP=1 BUILD_VER=v1.3.x
export BUILD_VER=v1.3.x
Env var set for BUILD_VER: v1.3.x
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.5.6
making output directory...
Building version v1.3.x
Document sets to generate:
scala_docs    : 1
java_docs     : 0
clojure_docs  : 1
doxygen_docs  : 1
r_docs        : 0
```

**Comment by stu1130** [comment]
@sandeep-krishnamurthy @nswamy could you merge this?

**Pull request merged**